### ISSUE TITLE
TTL for unknown entity

### ISSUE THREAD

**Issue by JasonLo** [UW-xDD/text2graph_llm#25]
In the full pipeline (search --> ask-xdd results --> llm graph extract --> align), we will encounter some unknown entity in the output, what the TTL format should look like for those object?

Example: 

```python

from text2graph.schema import Stratigraphy

unknown = Stratigraphy(strat_name="new_object")
unknown.model_dump()

{'strat_name': 'new_object',
 'strat_name_long': None,
 'rank': None,
 'strat_name_id': None,
 'concept_id': None,
 'bed': None,
 'bed_id': None,
 'mbr': None,
 'mbr_id': None,
 'fm': None,
 'fm_id': None,
 'subgp': None,
 'subgp_id': None,
 'gp': None,
 'gp_id': None,
 'sgp': None,
 'sgp_id': None,
 'b_age': None,
 't_age': None,
 'b_period': None,
 't_period': None,
 'c_interval': None,
 't_units': None,
 'ref_id': None}
```

**Comment by JasonLo** [comment]
`text2graph.gkm.triplet_to_rdf`  should able to handle these unknown object.

**Comment by JasonLo** [comment]
Resolved in #27
